src/App/ViewModels/JobDetailViewModel.cs

```diff
@@ -10,19 +10,50 @@
 using Prism.Commands;
 using Prism.Navigation;
 using PropertyChanged;
+using Xamarin.Essentials;
+using System.Windows.Input;
 
 namespace App.ViewModels
 {
     [AddINotifyPropertyChangedInterface]
-    public class JobDetailViewModel : IInitialize, INavigationAware
+    public class JobDetailViewModel : IInitialize
     {
+        private string uri;
+
         public JobDetailModel JobsDetail { get; set; }
-        public DelegateCommand OpenJobWebURLCommand { get; set; }
+
+        public DelegateCommand NavigateToURLCommand { get; set; }
 
         public JobDetailViewModel()
         {
 
+            NavigateToURLCommand = new DelegateCommand(async () => await NavigateToURL());
+
+        }
+
+        private async Task NavigateToURL()
+        {
+            var url = JobsDetail.Company.Url.ToString();
+            try
+            {
+
+                await Browser.OpenAsync(url, new BrowserLaunchOptions
+                {
+                    LaunchMode = BrowserLaunchMode.SystemPreferred,
+                    TitleMode = BrowserTitleMode.Show,
+                    PreferredToolbarColor = Color.AliceBlue,
+                    PreferredControlColor = Color.Violet
+                });
+
+            }
+            catch (Exception ex)
```

Comment:
Es posible mostrar un mensaje PopUp con la libreria que tu usas de mensajes, para cuando capture la excepcion, le muestre lo que esta sucediendo al cliente?  seria algo como por ejemplo, lo mismo que tienes comentado 

> "No Browser Application available to open"

Reply:
Si, se podría agregar ese Alert, además cuando implemente en otro feature los crashes de AppCenter

Reply:
![Alert-Navegador](https://user-images.githubusercontent.com/7376295/101658759-15912700-3a24-11eb-885a-9d04a9e00fc3.PNG)

Se utilizó Using the Page Dialog Service
https://prismlibrary.com/docs/xamarin-forms/dialogs/page-dialog-service.html

Reply:
Bien, aprobado